lte/gateway/python/integ_tests/s1aptests/BUILD.bazel

```diff
@@ -1119,6 +1181,24 @@ pytest_test(
     deps = [":s1ap_wrapper"],
 )
 
+pytest_test(
+    name = "test_enable_ipv6_iface",
+    size = "small",
+    srcs = ["test_enable_ipv6_iface.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_PRECOMMIT_TEST,
```

Comment:
We should add new tags to the file `bazel/test_constants.bzl`:
```
TAG_IPV6_TEST_ENABLE = ["ipv6_enable"] + TAG_MANUAL
TAG_IPV6_TEST_DISABLE = ["ipv6_disable"] + TAG_MANUAL
```
And tag the tests with these labels.
Even if it does not seem to break something, we should not run them out of order with the other precommit tests.

Reply:
discussed offline - remove from BUILD.bazel and add to check again.
* these "tests" are for setup and do not test something currently
* we will ask the owner if this can be included in specific setup/teardown steps in tests